1-components/1-shop-card/src/app/shop-card/shop-card.component.html

```diff
@@ -1,35 +1,35 @@
 <div class="product-list-box -basket">
   <div class="inner-col description-col">
     <div class="product-img">
-      <img/>
+      <img [src]="product?.image" [alt]="product.name" />
     </div>
     <div class="product-desc">
-      <h4 class="col-title mb-2"></h4>
+      <h4 class="col-title mb-2">{{product?.name}}</h4>
       <div class="rate">
-        <span class="rate-amount ml-2 d-inline-block d-md-none"> отзывов</span>
+        <span class="rate-amount ml-2 d-inline-block d-md-none">{{product?.feedbacksCount}} отзывов</span>
```

Comment:
```suggestion
        <span class="rate-amount ml-2 d-inline-block d-md-none">
             {{product?.feedbacksCount}} отзывов
        </span>
```
Общий кейс - лучше отделять контент от тегов, так читается лучше, когда все идет по 1 правилу: внутри тега - контент (edited)